test/e2e/storage/dax/dax.go

```diff
@@ -359,3 +361,130 @@ func testDaxOutside(
 	err := e2epod.DeletePodWithWait(f.ClientSet, pod)
 	framework.ExpectNoError(err, "while deleting pod")
 }
+
+// Hugepage page fault testing part starts here
+const (
+	accessHugepagesBinary = "_work/pmem-access-hugepages"
+)
+
+var _ = deploy.DescribeForSome("dax", func(d *deploy.Deployment) bool {
+	// Run these tests for all driver deployments that were created
+	// through the operator, where device mode is Direct.
+	return !d.HasOperator && d.HasDriver && d.Mode == api.DeviceModeDirect
+}, func(d *deploy.Deployment) {
+	var l local
+	f := framework.NewDefaultFramework("dax")
+	init := func() {
+		l = local{}
+
+		// Build pmem-access-hugepages helper binary.
+		l.root = os.Getenv("REPO_ROOT")
+		build := exec.Command("/bin/sh", "-c", os.Getenv("GO")+" build -o "+accessHugepagesBinary+" ./test/cmd/pmem-access-hugepages")
+		build.Stdout = GinkgoWriter
+		build.Stderr = GinkgoWriter
+		build.Dir = l.root
+		By("Compiling with: " + strings.Join(build.Args, " "))
+		err := build.Run()
+		framework.ExpectNoError(err, "compile ./test/cmd/pmem-access-hugepages")
+	}
+
+	config := &testsuites.PerTestConfig{
+		Driver:    nil,
+		Prefix:    "pmem",
+		Framework: f,
+	}
+	fstype := ""
+	vsource := v1.VolumeSource{
+		CSI: &v1.CSIVolumeSource{
+			Driver: "pmem-csi.intel.com",
+			FSType: &fstype,
+			VolumeAttributes: map[string]string{
+				"size": "110Mi",
+			},
+		},
+	}
+	It("should cause hugepage paging event with default fs", func() {
+		init()
+		testHugepageInPod(f, l.root, &vsource, config)
+	})
+	/* there is issue in kubelet causing panic and retry loop when fsType is set to ext4 or xfs.
```

Comment:
This now mentions the issue in kubelet causing panic and retries in pod creation (panic trace in kubelet log on corresponding worker). We should probably take that up on kubelet side.

Reply:
Please file an issue in kubernetes/kubernetes and link to it here.

Reply:
Ping?

Reply:
I filed an issue two days ago as I planned, the link is 2 lines below. Did I create issue in correct location?